spec/adapters/vertica_spec.rb

```diff
@@ -177,13 +177,34 @@ def VERTICA_DB.sqls
   end
 
   describe "Verticas's EXPLAIN and EXPLAIN LOCAL" do
-    specify "should not raise errors" do
+    specify "do not raise errors" do
```

Comment:
don't mind this changes but they should be on a separate commit/PR, there is no reason for a commit adding support for something new, to also have this misc fixes.